Snakefile

```diff
@@ -51,8 +57,7 @@ rule clingen:
         """
         python modules/ClinGen.py \
         --input_file {input} \
-        --output_file {output.evidenceFile} \
-        --unmapped_diseases_file {output.unmappedDiseases}
```

Comment:
I'm not sure we necessarily want to remove this output: unmapped diseases can be useful for informing the manual curation efforts.

However, it's not a problem for now. I can reimplement output of the unmapped diseases in a consistent way for all parsers at some point in the future.

Reply:
Ah, of course, makes sense! I do feel silly now :)